### PR TITLE
Add a Hot Deploy paragraph into README.md

### PR DESCRIPTION
**Pull request by mnagy**
@jhadvig please review?

### CODE REVIEW

**Comment by mnagy** [comment]
The test failure looks like a flake. It obviously couldn't have been caused by this PR, since it's only modifying a README.md

File: README.md [hunk 1]
```diff
@@ -273,3 +273,24 @@ following ways, in precedence order:
   in the case where you specify a path to a Python script via the `APP_FILE` environment
   variable, defaulting to a file named `app.py` if it exists. The script is
   passed to a regular Python interpreter to launch your application.
+
+Hot deploy
+---------------------
+
+If you are using Django, hot deploy will work out of the box.
```
Comment (jhadvig) [inline README.md]: Is the Gunicorn config file has set  `reload` option to true by default? And the `APP_CONFIG` env var is s pointing to the right config by default ?
If not nuke this line, cause thne the hot deploy is not working out of the box  
-

Comment (mnagy) [inline README.md]: No and no, but this is only applicable to running with Django, not Gunicorn. Django is only intended to run as a development web server, see: https://github.com/openshift/sti-python/blob/master/3.4/s2i/bin/run#L48 and https://docs.djangoproject.com/en/1.8/ref/django-admin/#runserver-port-or-address-port

Comment (jhadvig) [inline README.md]: But does that imply that the hot deploy works out of the box? That no action need to be done so user can modify his code and see immediately the change?

Comment (mnagy) [inline README.md]: Yes.

Comment (jhadvig) [inline README.md]: Yeah I just like the fact that first you are declaring that it's works OOTB and then then you actually tell to the user how to enable it for Gunicorn.

Comment (mnagy) [inline README.md]: Well, it says that it works OOTB *if* you use Django.

Comment (jhadvig) [inline README.md]: Well I can probably live with that

**Comment by jhadvig** [comment]
LGTM

**Pull request merged**